### DISCUSSION TITLE
Where's the Mastodon Dev Roadmap?

### DISCUSSION THREAD

**Discussion by steevithak** [mastodon/mastodon#22644] [General]
I'm a new Mastodon user and exploring the source code. Can someone point me to Mastodon's development roadmap that shows what features are planned for upcoming versions and what work needs to be done for each release?

**Comment by thefuturebird** [comment]
This would be very helpful. I've been watching this post hoping for a response.

**Comment by andreykurenkov** [comment]
Same here, hoping to see this. Having this would go a long way toward addressing my questions in this discussion: https://github.com/mastodon/mastodon/discussions/22428

**Comment by Gargron** [comment]
New: https://joinmastodon.org/roadmap

**Reply by steevithak** [reply]
Thanks!

**Reply by marek-saji** [reply]
Awesome to see this, but a shame that overflowing titles are truncated. I can see only e.g. “MAS-35 Rename "Everything" in O…”

**Comment by andreykurenkov** [comment]
Awesome, thanks!
